script/texdoclib-const.tlu

```diff
@@ -25,43 +25,147 @@ progname = 'Texdoc'
 version = '3.4.1'
 release_date = '2022-03-19'
 
--- make sure to update setup_config_from_cl() accordingly
--- and set a default value in setup_config_from_defaults() if relevant
-usage_msg = [[
-Usage: texdoc [OPTION]... NAME...
-   or: texdoc [OPTION]... ACTION
-
-Try to find appropriate TeX documentation for the specified NAME(s).
-Alternatively, perform the given ACTION and exit.
-
-Options:
-  -w, --view        Use view mode: start a viewer. (default)
-  -m, --mixed       Use mixed mode (view or list).
-  -l, --list        Use list mode: show a list of results.
-  -s, --showall     Use showall mode: show also "bad" results.
-
-  -i, --interact    Use interactive menus. (default)
-  -I, --nointeract  Use plain lists, no interaction required.
-  -M, --machine     Machine-readable output for lists (implies -I).
-
-  -q, --quiet       Suppress warnings and most error messages.
-  -v, --verbose     Print additional information (e.g., viewer command).
-  -D, --debug       Activate all debug output (equal to "--debug=all").
-  -d LIST, --debug=LIST
-                    Activate debug output restricted to LIST.
-  -c NAME=VALUE     Set configuration item NAME to VALUE.
-
-Actions:
-  -h, --help        Print this help message.
-  -V, --version     Print the version number.
-  -f, --files       Print the list of configuration files used.
-  --just-view FILE  Display FILE, given with full path (no searching).
-
-Full manual available via `texdoc texdoc'.
-
-Website: <https://tug.org/texdoc/>
-Repository: <https://github.com/TeX-Live/texdoc>
-Please email bugs to <texdoc@tug.org>.]]
+function get_boolean_action(key, val)
```

Comment:
Please do not include any function definition in `texdoclib-const.tlu`. This can be moved to `texdoclib-cli.tlu`.